src/mount/writedata.cc

```diff
@@ -596,36 +678,25 @@ void InodeChunkWriter::processDataChain(ChunkWriter& writer) {
 		}
 
 		// Let's sleep a bit shorter if we can't be woken up by the pipe to reduce the latency
-		writer.processOperations(inodeData_->isDataChainPipeValid() ? 50 : 10);
+		writer.processOperations(chunkData_->isDataChainPipeValid() ? 50 : 10);
```

Comment:
Consider the substitution of magic values 50 and 10 here by constants, in case at some point it could be necessary to change these values.

Reply:
Done.